client/dashboard/components/settings/general/store-address.tsx

```diff
@@ -244,21 +306,31 @@ export function useGetCountryStateAutofill( options, countryState, setValue ) {
 				name="state"
 				type="text"
 				className="woocommerce-select-control__autofill-input"
-				tabIndex="-1"
+				tabIndex={ -1 }
 				autoComplete="address-level1"
 			/>
 		</>
 	);
 }
 
+type StoreAddressProps = {
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Is there a way to avoid this?

Reply:
I think `any` is best for now because we don't have the defined type for `getInputProps` from the source (`component/From` still uses JavaScript only). And we cannot use `unknown` here since many variables are derived from it.